### DISCUSSION TITLE
Can the config.userSelect property be set for a Touchable component?

### DISCUSSION THREAD

**Discussion by km-tr** [software-mansion/react-native-gesture-handler#2799] [Q&A] [answered]
I wanted to set the userSelect property on a TouchableOpacity component, but it seems to automatically default to 'none'.

I am considering the scenario where the code is running on the web.

I thought the following might be related to this issue:

- https://github.com/software-mansion/react-native-gesture-handler/blob/1f723ecfc726e7300a2bd09957170332bb1c56c9/src/web/tools/GestureHandlerWebDelegate.ts#L39-L41
- https://github.com/software-mansion/react-native-gesture-handler/blob/1f723ecfc726e7300a2bd09957170332bb1c56c9/src/handlers/NativeViewGestureHandler.ts#L46-L54

Is it possible to set the userSelect property in this config from within the TouchableOpacity component?

**Comment by km-tr** [comment]
@m-bert 
Excuse me, what do you think about this?

**Reply by m-bert** [reply, marked as answer]
You're right, currently it is not possible to apply this property. However, [this PR](https://github.com/software-mansion/react-native-gesture-handler/pull/2812) makes it possible!
